bodhi/server/config.py

```diff
@@ -483,7 +483,7 @@ class BodhiConfig(dict):
             'value': 'https://admin.fedoraproject.org/pkgdb',
             'validator': unicode},
         'pkgtags_url': {
-            'value': 'https://apps.fedoraproject.org/tagger/api/v1/tag/sqlitebuildtags/',
+            'value': '',
```

Comment:
Ah, this was definitely a mistake. I had seen that this value was commented in [production.ini](https://github.com/fedora-infra/bodhi/blob/2.7.0/production.ini#L266-L268):

```
# We used to get our package tags from pkgdb, but they come from tagger now.
# https://github.com/fedora-infra/fedora-tagger/pull/74
#pkgtags_url = https://apps.fedoraproject.org/tagger/api/v1/tag/sqlitebuildtags/
```

and mistakenly made it the default. This probably happened as a result of how I made the change - I had been going through the file and the code for each setting, and as I encoded each setting I also made the default be in the ```production.ini``` file commented like the above. My guess is that I saw that commented value and assumed that I commented it, and then put the value to match it. D'oh.

Thanks so much for figuring this out and for fixing it. You are a huge help!